src/tools/lint-docs/src/lib.rs

```diff
@@ -444,6 +444,7 @@ impl<'a> LintExtractor<'a> {
         let mut cmd = Command::new(self.rustc_path);
         if options.contains(&"edition2024") {
             cmd.arg("--edition=2024");
+            cmd.arg("-Zunstable-options");
```

Comment:
Question [REV3,ED2024]: if this is no longer edition gated, I don't think having unstable options here is correct.